src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -85,9 +85,10 @@ internal enum TerminatorState
             IsEndOfFunctionPointerParameterListErrored = 1 << 24,
             IsEndOfFunctionPointerCallingConvention = 1 << 25,
             IsEndOfRecordOrClassOrStructOrInterfaceSignature = 1 << 26,
+            IsExpressionOrPatternInCaseLabelOfSwitchStatement = 1 << 27
```

Comment:
```suggestion
            IsExpressionOrPatternInCaseLabelOfSwitchStatement = 1 << 27,
```

include a trailing comma to be consistent with this declaration before the change.